### PR TITLE
[Bug] Fix reset not working (#948)

### PR DESCRIPTION
**Pull request by llanghammer-sap**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) (if applicable)
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

## Description
Fixes the bug that that the Reset Button is not doing anything when pressed and changes the reset function to call store.clearAll() to remove all locally saved items from the secureStore

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/SceneDelegate.swift [hunk 1]
```diff
@@ -297,10 +297,11 @@ extension SceneDelegate: ENAExposureManagerObserver {
 }
 
 extension SceneDelegate: HomeViewControllerDelegate {
+	/// Resets all stores and notifies the Onboarding.
 	func homeViewControllerUserDidRequestReset(_: HomeViewController) {
-		store.isOnboarded = false
-		store.dateLastExposureDetection = nil
+		store.clearAll()
 		UIApplication.coronaWarnDelegate().downloadedPackagesStore.reset()
```
Comment (haosap) [inline src/xcode/ENA/ENA/Source/SceneDelegate.swift:303]: good point, I forgot to tell you to remove downloadedPackagesStore.reset().

**Review by haosap — APPROVED** [review]

**Comment by haosap** [comment]
Hi @AdnanZiSAP ,
Please review it. Thanks.

File: src/xcode/ENA/ENA/Source/SceneDelegate.swift (hunk 1 above)
Comment (llanghammer-sap) [reply src/xcode/ENA/ENA/Source/SceneDelegate.swift]: Removed with https://github.com/corona-warn-app/cwa-app-ios/pull/168/commits/24d1f9fa749014f5f8d412af17e7ab508604d9cb

File: src/xcode/ENA/ENA/Source/Scenes/Settings/ResetViewController.swift [hunk 2]
```diff
@@ -36,6 +36,7 @@ final class ResetViewController: UIViewController {
 
 	@IBAction func resetData(_: Any) {
 		delegate?.reset()
+		dismiss(animated: true, completion: nil)
```
Comment (AdnanZiSAP) [inline src/xcode/ENA/ENA/Source/Scenes/Settings/ResetViewController.swift:39]: After the reset this dismissing shouldn't be needed.

Comment (llanghammer-sap) [reply src/xcode/ENA/ENA/Source/Scenes/Settings/ResetViewController.swift:39]: If i remove it the Reset screen is not automatically dismissed and stays on top of the Onboarding Workflow until it is dismissed manually. 
This happens on all of my test devices so I would leave it in.

Comment (AdnanZiSAP) [reply src/xcode/ENA/ENA/Source/Scenes/Settings/ResetViewController.swift:39]: Yes, that is what I noticed too.

**Review by AdnanZiSAP — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/SceneDelegate.swift (hunk 1 above)
Comment (haosap) [reply src/xcode/ENA/ENA/Source/SceneDelegate.swift]: Hi @llanghammer-sap ,
Sorry, I think we have misunderstood here. :-)
We need to remove the downloadPakcageStore.
Thanks.

**Pull request merged**